rust/connlib/tunnel/src/lib.rs

```diff
@@ -261,6 +262,10 @@ pub enum ClientEvent {
     ResourcesChanged {
         resources: Vec<callbacks::ResourceDescription>,
     },
+    DnsServersChanged {
+        /// The map of DNS servers that connlib will use, indexed by the proxy IP assigned by connlib.
```

Comment:
```suggestion
        /// The map of DNS servers that connlib will use, indexed by the proxy (or "sentinel") IP assigned by connlib.
```
I've just been calling them sentinel IPs and sentinel resolvers interchangeably, though I can't find anywhere else online that calls them sentinels